nextcloud_update.sh

```diff
@@ -324,13 +324,41 @@ then
     check_command phpenmod -v ALL redis
 fi
 
-# Upgrade APCu and igbinary
-#if [ "${CURRENTVERSION%%.*}" -ge "17" ]
-if [ "${CURRENTVERSION%%.*}" -ge "23" ]
+# Change from APCu to Redis for local cache
+if pecl list | grep redis >/dev/null 2>&1
+then
+    nextcloud_occ config:system:set memcache.local --value='\OC\Memcache\Redis'
```

Comment:
@szaimen

Reply:
ah sorry, didn't see this.
But shouldn't more likely be called
`nextcloud_occ config:system:set memcache.local --value='\\OC\\Memcache\\Redis'`

(notice the double slashes)

Reply:
Nope :)